### DISCUSSION TITLE
How to see User Device Activity in Keycloak Admin Console?

### DISCUSSION THREAD

**Discussion by kresnasatya** [keycloak/keycloak#21054] [Q&A]
Currrently, Device Activity available in Keycloak Account Console not in Keycloak Admin Console. Is that Keycloak support to allow administrator to see device activity that user used to login with Keycloak Platform in Keycloak Admin Console? I check in Keycloak database there's no table related with this case. Even though Keycloak has table like user_session but it doesn't show device activity information such as browser, desktop or mobile platform that user use (eg. Android or iOS or Windows or Linux, etc).

**Comment by abstractj** [comment]
@kresnasatya as far as I'm aware, device activity is available in the Keycloak Account Console but not in the Keycloak Admin Console.

**Reply by kresnasatya** [reply]
Is that Keycloak have a plan to ship this feature on their Admin Console? Or at least ship in Keycloak Admin REST API. So far I know that to get device activity of user used, I must use their access token (I inspect element and check request url to get device activity of user used).
